### PR TITLE
Handle 'message' events

### PR DESCRIPTION
**Pull request by andrefgneves**
This adds a new `MessageEvent` model that is used to handle events of type `message`. These events can be triggered on a worker by using the workers `postMessage` API.

### CODE REVIEW

File: packages/service-worker-mock/models/MessageEvent.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const ExtendableEvent = require('./ExtendableEvent');
+
+class MessageEvent extends ExtendableEvent {
+  constructor(args) {
```
Comment (zackargyle) [inline packages/service-worker-mock/models/MessageEvent.js]: Can we have this match the API described here? [Link](https://developer.mozilla.org/en-US/docs/Web/API/MessageEvent/MessageEvent)

```js
new Message(type: string, { data?, origin?, lastEventId?, source?, ports? });
```

Comment (andrefgneves) [reply packages/service-worker-mock/models/MessageEvent.js]: Sure thing.

**Comment by andrefgneves** [comment]
@zackargyle any news on this?

**Comment by zackargyle** [comment]
Ah yes, thanks for the ping. Looks good to me! Merging now.

**Comment by zackargyle** [comment]
Pushed new version to 1.8.0 👍🏻

**Comment by andrefgneves** [comment]
Awesome, thanks

**Pull request merged**